### PR TITLE
#162948883 Users should be able to follow each other

### PR DESCRIPTION
**Pull request by boltonotieno**
#### What does this PR do?
* Users should be able to follow and or unfollow other users.

#### Description of Task(s) to be completed?
* Create a follow button component
* Link follow button on user profile
* Add following and followers count
* Create a view for user followers and following

#### How should this be manually tested?
* Clone the repo `git clone https://github.com/andela/ah-alpha-frontend.git`
* Checkout to the branch `git checkout ft-follow-unfollow-user-162948883`
* Install dependencies `npm install`
* Run the server `npm start`

### Screenshots
* Profile
image

* Following
image

* Followers
image
 
#### What are the relevant pivotal tracker stories?
[#162948883](https://www.pivotaltracker.com/story/show/162948883)

### CODE REVIEW

File: src/actions/profileActions.js [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+export const updateProfile = (username, data) => (dispatch) => {
+  profileApi.updateProfile(username, data)
+    .then((res) => {
+      if (res.success) {
+        dispatch(updateProfileSuccess(res.content.profile[0]));
+        toast.success("Profile Updated Successfully");
+        // eslint-disable-next-line func-names
+        // eslint-disable-next-line no-restricted-globals
+        setTimeout(() => { location.href = "/profile"; }, 2500);
+      // window.location.href = "/profile";
```
Comment (danuluma) [inline src/actions/profileActions.js]: Please remove commented code

File: src/api/axiosConfig.js [hunk 2]
```diff
@@ -0,0 +1,18 @@
+import axios from "axios";
+
+const token = localStorage.getItem("token");
+
+// const urlPath = axios.create({
+//   baseURL: process.env.BASE_URL,
+
+// const token = localStorage.getItem("token");
```
Comment (danuluma) [inline src/api/axiosConfig.js]: Remove commented code here too

**Review by Ruiru11 — APPROVED** [review]
Good work

**Review by Deekerubo — COMMENTED** [review]
Remove the source.env file

**Review by danuluma — APPROVED** [review]

**Comment by boltonotieno** [comment]
> Remove the source.env file

Thank you for the feedback, this has been worked on

**Review by Betty-Kebenei — APPROVED** [review]

**Pull request merged**